tests/plcc/compiles-obj/OBJ/grammar

```diff
@@ -157,3 +157,4 @@ include val
 include list
 include class
 include ref
+%
```

Comment:
We should leave this alone as it tests a traditional 3-section grammar. If you modify this, then we don't know if your changes break what was working before.

Instead, we need a separate test for a 4-section grammar. You could copy "compiles-obj" to something like "compiles-4-section-grammar", and then make this change.